tests/components/shelly/test_config_flow.py

```diff
@@ -244,8 +337,43 @@ async def test_zeroconf_require_auth(hass):
             data={"host": "1.1.1.1", "name": "shelly1pm-12345"},
             context={"source": config_entries.SOURCE_ZEROCONF},
         )
-        assert result["type"] == "abort"
-        assert result["reason"] == "auth_not_supported"
+        assert result["type"] == "form"
+        assert result["errors"] == {}
+
+    result2 = await hass.config_entries.flow.async_configure(
+        result["flow_id"],
+        {},
+    )
+    assert result2["type"] == "form"
+    assert result2["errors"] == {}
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6103615)